src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -165,122 +163,21 @@ private void HandleClusterDatabaseChanged(string databaseName, long index, strin
             }
         }
 
-        private void NotifyPendingClusterTransaction(string databaseName, Task<DocumentDatabase> task, long index)
+        public void NotifyPendingClusterTransaction(string name, Task<DocumentDatabase> task)
```

Comment:
Given that we _know_ that the tx is completed here, why are getting as `Task` instead of the `DocumentDatabase` ?

Reply:
Simply to have same behavior as `NotifyDatabaseAboutStateChange` and `NotifyDatabaseAboutValueChange`.

Reply:
They should be changed to avoid this as well, yes.